src/mmw/apps/user/views.py

```diff
@@ -93,10 +93,36 @@ def itsi_auth(request):
     else:
         # User did not authenticate. Save their ITSI ID and send to /register
         request.session['itsi_id'] = itsi_user['id']
-        return redirect('/register')  # TODO Make this Backbone route
+        return redirect('/register/{username}/{first_name}/{last_name}'.format(
+            **itsi_user['extra']
```

Comment:
Is this an ITSI-ism? Seems strange to have username, first name, and last name in a URL.